docs/examples/config

```diff
@@ -11,6 +11,7 @@ poll_method		epoll		# poll, select, epoll ..
 #sip_listen		0.0.0.0:5060
 #sip_certificate	cert.pem
 #sip_cafile		ca.crt
+#sip_trans_def		udp
```

Comment:
each user-agent has a preferable sip transport.

it is possible to re-use each user-agents transport instead of adding a global one?
if possible, this will reduce the complexity.

Example:

```
<sip:user@domain.com>
<sip:user@domain.com;transport=tcp>
<sip:user@domain.com;transport=tls>
```

Reply:
We have the account on the one side and the peer URI on the other side. Then we try to match these.

In uri_match_transport() we use this config as default value for the peer-uri. Not for the account.

To simplify this we would have to suppose "UDP" always. Then the application/user will have to add always the URI parameter "transport" to the target URI if something different is wanted. Do you prefer this?